run/createRunDir.sh

```diff
@@ -294,6 +294,7 @@ mkdir -p ${rundir}
 
 # Copy run directory files and subdirectories
 cp -r ./OutputDir ${rundir}
+cp -r ./OutputDir ${rundir}/Restarts
```

Comment:
Could we change this to `mkdir -p ${rundir}/Restarts` for consistency with createRunDir.sh for GCClassic?

Reply:
@msulprizio: Now fixed in commit ff5adb9

Reply:
Now resolved by https://github.com/geoschem/HEMCO/pull/202/commits/ff5adb9fa71274165c174b469f9c45d7442c0a68